packages/frontend/src/main/waggleManager.ts

```diff
@@ -59,4 +62,6 @@ export const runWaggle = async (
 
 export const waggleVersion = waggle.version
 
-export default { getPorts, runWaggle, waggleVersion }
+runWaggle().catch((e) => {
+  log.error('Waggle initialization returned error', e)
```

Comment:
I think we should start to either throwing errors or when it is possible handle them. This will add necessary criticism, errors will be more visible to us. For example this one won't be obvious until you start thinking that something is wrong because spinner spins for a longer time than usual. It will also give us much better feedback from Kinga and Holmes.

Reply:
I've created general issue for that; https://github.com/ZbayApp/monorepo/issues/446

Reply:
I added throwing the error.